frontend/apps/reader/readerui.lua

```diff
@@ -329,6 +330,7 @@ end
 
 function ReaderUI:showReader(file)
     DEBUG("show reader ui")
+    ReadHistory:addItem(file)
```

Comment:
I would even say go with 500 ;) I double anyone would want to keep track 500 books at once. But if you can test on the device and double check there is not a huge performance difference between 1000 and 500, then 1000 works for me too.